core/chainio/avs_writer.go

```diff
@@ -93,39 +94,78 @@ func (w *AvsWriter) SendAggregatedResponse(batchIdentifierHash [32]byte, batchMe
 	txOpts.NoSend = false
 	i := 0
 
+	var sentTxs []*types.Transaction
+
+	batchMerkleRootHashString := hex.EncodeToString(batchMerkleRoot[:])
+
 	respondToTaskV2Func := func() (*types.Receipt, error) {
 		gasPrice, err := utils.GetGasPriceRetryable(w.Client, w.ClientFallback)
 		if err != nil {
 			return nil, err
 		}
-
-		bumpedGasPrice := utils.CalculateGasPriceBumpBasedOnRetry(gasPrice, gasBumpPercentage, gasBumpIncrementalPercentage, i)
-		// new bumped gas price must be higher than the last one (this should hardly ever happen though)
-		if bumpedGasPrice.Cmp(txOpts.GasPrice) > 0 {
-			txOpts.GasPrice = bumpedGasPrice
+		previousTxGasPrice := txOpts.GasPrice
+		// minimum gas price required for the new transaction to ensure it's 10% higher than the previous one.
```

Comment:
Is this outdated?

Reply:
No, it was not, but the comment was a little confusing. I've updated it, it should be clearer now.